### ISSUE TITLE
Ping timeout

### ISSUE THREAD

**Issue by bidak** [vysheng/tg#760]
I want to create multiple daemon files with following code:
```
../bin/telegram-cli -k tg-server.pub -c telegram.config --json -dWS /var/www/tg1.sck &
...
...
```
But telegram-cli shows ping timeout and not connected.
There is any way to solve the problem?

**Comment by LukeLR** [comment]
Does it work with only one client?

**Comment by bidak** [comment]
Yes, It works with one client.
When the connection is lost, I am use the following code:
```
sudo pkill telegram
```
So tg connects.

**Comment by bidak** [comment]
How to solve this problem?

**Comment by LukeLR** [comment]
So when you're launching a single instance of tg, everything works and you can send a message to someone?

**Comment by bidak** [comment]
Yes , i'm connecting and sending message now but when i create multiple deamon tg , shows fail connection.

**Comment by LukeLR** [comment]
Are you sure that all the instances are using different configuration directories?

**Comment by bidak** [comment]
Thanks a lot.this problem has been solved.
Is there any way to find out if the account is blocked? (without terminal and by json or ...)
